icubam/analytics/data.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+
+from absl import logging  # noqa: F401
+
+from icubam.analytics import preprocessing
+from icubam.db import store
+
+BASE_PATH = Path(__file__).resolve().parent
+
+CUM_COLUMNS = [
+  "n_covid_deaths",
+  "n_covid_healed",
+  "n_covid_transfered",
+  "n_covid_refused",
+]
+NCUM_COLUMNS = [
+  "n_covid_free",
+  "n_ncovid_free",
+  "n_covid_occ",
+  "n_ncovid_occ",
+]
+BEDCOUNT_COLUMNS = CUM_COLUMNS + NCUM_COLUMNS
+ALL_COLUMNS = ([
+  "icu_name", "date", "datetime", "department", "region_id", "region",
+  "create_date"
+] + CUM_COLUMNS + NCUM_COLUMNS)
+
+
+def load_bed_counts(db, preprocess: bool = False):
+  """Load Bedcount data from ICUBAM DB"""
+  bc = store.to_pandas(db.get_bed_counts(), max_depth=2)
+  if preprocess:
+    bc = preprocessing.preprocess_bedcounts(bc)
```

Comment:
I think it's better to explicitly apply processioning function after the load. Among other things because there are multiple way to do preprocessing with potentially different parameters. Users of this module should read `preprocess_bedcounts` docstring instead of just applying `preprocess=True` here which obfuscates it.

Reply:
Done